### DISCUSSION TITLE
Add additional attendees to existing event

### DISCUSSION THREAD

**Discussion by amit-soni7** [spatie/laravel-google-calendar#248] [Ideas]
$event->addAttendee() method is available to add attendees to an event. however we want to add additional email/ attendees to same event, it remove previously added attendees.

for example:
add a@example.com and b@example.com and save();

if we want add c@example.com in update/patch, it removes previously added a and b then adds c.

what can we do with this situation??

**Comment by freekmurze** [comment]
This is currently to possible. Feel free to PR this.

**Reply by yudapratama25** [reply]
i have the same problem currently in 2023, please help

**Reply by amit-soni7** [reply]
Use my PR for the same.
